x-pack/plugins/fleet/server/services/epm/elasticsearch/template/install.test.ts

```diff
@@ -121,71 +81,11 @@ describe('EPM install', () => {
     };
     const templateIndexPatternDatasetIsPrefixTrue = 'metrics-package.dataset.*-*';
     const templatePriorityDatasetIsPrefixTrue = 150;
-    await installTemplate({
-      esClient,
-      logger: loggerMock.create(),
-      fields,
-      dataStream: dataStreamDatasetIsPrefixTrue,
-      packageVersion: pkg.version,
-      packageName: pkg.name,
-    });
-
-    const sentTemplate = (
-      esClient.indices.putIndexTemplate.mock.calls[0][0] as estypes.IndicesPutIndexTemplateRequest
-    ).body;
-
-    expect(sentTemplate).toBeDefined();
-    expect(sentTemplate?.priority).toBe(templatePriorityDatasetIsPrefixTrue);
-    expect(sentTemplate?.index_patterns).toEqual([templateIndexPatternDatasetIsPrefixTrue]);
-  });
-
-  it('tests installPackage remove the aliases property if the property existed', async () => {
```

Comment:
I removed this whole test because this behavior is no longer necessary and was removed in the previous PR. This test just happens to still pass.